src/RichText/RichText.tsx

```diff
@@ -36,7 +41,12 @@ const DEV_SERVER_URL = 'http://localhost:3000';
 // TODO: make it a prop
 const TOOLBAR_HEIGHT = 44;
 
-export const RichText = ({ editor, ...props }: RichTextProps) => {
+export const RichText = ({
+  editor,
+  onMessage,
+  exclusivelyUseCustomOnMessage = true,
```

Comment:
What is the use-case for this?

Reply:
for the prop? Keeping compatibility and maintaining current behaviour so I don't accidentally break peoples' setups. And in general the use-case for the pr is being able to use the `onMessage` prop of the webview without messing up tentap's own behaviour

Reply:
Any case it's a very important feature for me, so I know there's a usecase haha